synfig-studio/src/gui/workarea.h

```diff
@@ -472,6 +475,9 @@ class WorkArea : public Gtk::Grid, public Duckmatic
 	bool on_hruler_event(GdkEvent* event);
 	bool on_vruler_event(GdkEvent* event);
 	void on_duck_selection_single(const etl::handle<Duck>& duck_guid);
+
+    // Sets the current cursor regardless of drag mode
+    void set_cursor_forced(const Glib::RefPtr<Gdk::Cursor> &x);
```

Comment:
fix indentation (we use tab, not spaces)

Reply:
done